compiler/rustc_resolve/src/late/lifetimes.rs

```diff
@@ -1,3 +1,4 @@
+// ignore-tidy-filelength
```

Comment:
Is this still needed now? If so, not a big deal.

Reply:
still needed